Foundation/NSByteCountFormatter.swift

```diff
@@ -47,48 +47,261 @@ open class ByteCountFormatter : Formatter {
     }
     
     public required init?(coder: NSCoder) {
-        NSUnimplemented()
+        super.init(coder: coder)
```

Comment:
We should probably leave this as unimplemented, because clearly the byte count formatter should be restoring settings like allowedUnits, countStyle, etc. from the archive.